package.json

```diff
@@ -30,18 +29,16 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/nestjs/nest-cli.git"
+    "url": "git+https://github.com/sooraj1002/stencil-cli.git"
```

Comment:
make this techsavvyash as it will be the base repository that will be moved around to orgs and actively maintained.